src/chainstate/stacks/db/blocks.rs

```diff
@@ -4859,6 +4869,135 @@ impl StacksChainState {
         Ok(parent_miner)
     }
 
+    fn get_stacking_and_transfer_burn_ops_v205(
+        sortdb_conn: &Connection,
+        burn_tip: &BurnchainHeaderHash,
+    ) -> Result<(Vec<StackStxOp>, Vec<TransferStxOp>), Error> {
+        let stacking_burn_ops = SortitionDB::get_stack_stx_ops(sortdb_conn, burn_tip)?;
+        let transfer_burn_ops = SortitionDB::get_transfer_stx_ops(sortdb_conn, burn_tip)?;
+        Ok((stacking_burn_ops, transfer_burn_ops))
+    }
+
+    fn get_stacking_and_transfer_burn_ops_v210<'b>(
+        chainstate_tx: &'b mut ChainstateTx,
```

Comment:
I don't think the lifetime generic is necessary here: rust can almost always figure out the lifetimes for references that don't land in the return type.